README.md

```diff
@@ -73,6 +73,23 @@ myPromise.then(function(val) {});
 myPromise.then(() => { doSomething(); });
 ```
 
+### `no-then-fail-param`
+
+Ensures that each `then()` function can only one parameter (the success handler). This avoids a [common anti-pattern](https://github.com/petkaantonov/bluebird/wiki/Promise-anti-patterns#the-thensuccess-fail-anti-pattern).
```

Comment:
Nit: values passed in a function call are _arguments_. The variables that receive the arguments in a function declaration are _parameters_. In this case, `then` is the function and you're talking about limiting the number of arguments you can give it.